angular-ngrx-scss/src/app/home/top-repositories/top-repositories.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Honestly think we can remove this test file - the component doesn't do any heavy logic so I don't think there's anything worth testing in here.

- [ ] Can remove this test file